pages/map.py

```diff
@@ -148,6 +196,15 @@ def create_user_emails_options(trips_group_by_user_id):
             options.append(create_single_option(user_email, color))
     return options, user_emails
 
+def create_user_modes_options(trips_group_by_user_mode):
+    options = list()
+    user_modes = set()
+    for user_mode in trips_group_by_user_mode:
+        color = trips_group_by_user_mode[user_mode]['color']
```

Comment:
I am looking at the implementation of `get_trips_by_user_mode`, and I don't see where the 'color' is filled in.
`trips_group_by_user_mode = trips_df.groupby('data.user_input.mode_confirm')` so unless `trips_df` has a `color`, `trips_group_by_user_mode` will not have a color.

Can you please clarify how this is supposed to work?